.editorconfig

```diff
@@ -0,0 +1,21 @@
+# editorconfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+indent_style = space
+indent_size = 2
+
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[{package.json}]
+indent_style = space
+indent_size = 2
```

Comment:
Just to remove some annoying tools complaints, I think it's better to add a new line at the end of the file.